### PR TITLE
Fix Typscript Version Warning

### PR DESCRIPTION
**Pull request by Kicchu02**
Fixed the following error.

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,14 +20,14 @@
     "react": "18.2.0",
     "react-native": "0.74.5",
     "react-native-gesture-handler": "~2.16.1",
-    "save-dev": "^0.0.1-security",
+    "react-native-safe-area-context": "4.10.5",
     "react-native-screens": "3.31.1",
-    "react-native-safe-area-context": "4.10.5"
+    "save-dev": "^0.0.1-security"
   },
```
Comment (Kicchu02) [inline package.json]: The values do not change here. Why were they re-arranged?

File: package-lock.json [hunk 2]
```diff
@@ -13965,10 +13965,11 @@
       }
     },
     "node_modules/typescript": {
-      "version": "5.5.4",
-      "resolved": "https://registry.npmjs.org/typescript/-/typescript-5.5.4.tgz",
```
Comment (Kicchu02) [inline package-lock.json]: If the latest version is 5.5.4, then why are we downgrading? Can't we update the in other places to use the latest typescript versions?

File: package.json (hunk 1 above)
Comment (Kicchu02) [reply package.json]: Reverted.

File: package-lock.json (hunk 2 above)
Comment (Kicchu02) [reply package-lock.json]: Updated.

**Review by law-levi-san — APPROVED** [review]
alright

**Review by law-levi-san — COMMENTED** [review]
why re-arrange in the first place?

**Comment by Kicchu02** [comment]
> why re-arrange in the first place?

Running the commands to update the version re-arranged the objects in the `package.json` by default. For now, I have manually reverted it back. But in future, if commands that involve modification of the `package.json` and `package-lock.json` to this re-arrangement again, then we just merge it.

**Pull request merged**